workers/questionreviewer/testhelpers/postgres_testcontainer.go

```
@@ -0,0 +1,102 @@
+		interval INT NOT NULL DEFAULT 1,
+		next_review_instant TIMESTAMP NOT NULL DEFAULT NOW(),
+		UNIQUE(question_id, user_id)
+	);
+	`
+	if _, err := dbConn.Exec(schema); err != nil {
+		t.Fatalf("could not init schema: %v", err)
+	}
+
+	teardown := func() {
+		dbConn.Close()
+		container.Terminate(ctx)
```

golangci-lint (1.64.8)

Error return value of `container.Terminate` is not checked

(errcheck)